pandas/io/json.py

```diff
@@ -204,6 +208,12 @@ def read_json(path_or_buf=None, orient=None, typ='frame', dtype=True,
     else:
         json = filepath_or_buffer
 
+    if lines:
+        # If given a json lines file, we break the string into lines, add
+        # commas and put it in a json list to make a valid json object.
+        lines = list(StringIO(json))
+        json = '[' + ','.join(lines) + ']'
```

Comment:
Yeah it is a problem, I didn't think about it. I'll see if I can find a
reliable way to do it in PY3